library/ord-map.lisp

```diff
@@ -174,7 +174,9 @@ If ITER contains duplicate keys, later values will overwrite earlier values."
                                                       (helper less)))
                    ((Eq) (Some (tree::Branch clr less (MapPair pivot (func val)) more)))
                    ((GT) (coalton-library/classes:map (tree::Branch clr less (MapPair pivot val))
-                                                      (helper more)))))))))
+                                                      (helper more)))))
+                (_
```

Comment:
I would change to `((tree::DoubleBlackEmpty) (error "Encountered double-black node in ordered map outside of removal operation"))`.